components/datetime/tests/fixtures/tests/components-partial-matches.json

```diff
@@ -11,13 +11,13 @@
         },
         "output": {
             "values": {
-                "en": "5:07",
-                "fr": "5:07"
+                "en": "8:5 AM",
+                "fr": "8:5 AM"
```

Comment:
`question`: is this something we ever want to see in `en` or `fr` or an artificial test?

Reply:
I filed https://github.com/unicode-org/icu4x/issues/1581

I don't think this needs to be fixed in this PR, but should be something to be fixed as a follow-up.